dev/deps/spark-deps-hadoop-2.7-hive-2.3

```diff
@@ -126,7 +126,7 @@ javax.inject/1//javax.inject-1.jar
 javax.jdo/3.2.0-m3//javax.jdo-3.2.0-m3.jar
 javax.servlet-api/3.1.0//javax.servlet-api-3.1.0.jar
 javolution/5.5.1//javolution-5.5.1.jar
-jaxb-api/2.2.2//jaxb-api-2.2.2.jar
+jaxb-api/2.2.11//jaxb-api-2.2.11.jar
```

Comment:
As we don't touch hadoop 2.7 deps, why do we need to change this file?

Reply:
This is a side-effect from the default Hadoop 3.x profile: since the shaded client jars exclude 3rd party dependencies include this, we'd have to explicitly import it even for 2.x profile. With some Maven trick maybe we can avoid the change but I'm not sure if this is a big deal.

Reply:
It's not a concern, and I just want to check why this is changed.